docs/docs/user-guide/using-cli.md

```diff
@@ -75,10 +75,10 @@ To check if CLI detected the SPO URL previously, use the `m365 spo get` command.
 
 ## Passing complex content into CLI options
 
-When passing complex content into CLI options, such as JSON strings, you will need to properly escape nested quotes. The exact way to do it, depends on the shell that you're using. Alternatively, you can choose to pass complex content by storing the complex content in a file and passing the path to the file prefixed with an `@`, for example:
+When passing complex content into CLI options, such as JSON strings, you will need to properly escape nested quotes. The exact way to do it, depends on the shell that you're using. Alternatively, you can choose to pass complex content by storing the complex content in a file and passing the path to the file prefixed with an `@` that needs to be escaped in PowerShell, for example:
```

Comment:
I think we may do something like this
```
`` `@``
```
so it will render to something like this 
![image](https://user-images.githubusercontent.com/58668583/168684425-df7277bb-fb44-42ea-afd2-39f2acfdab6f.png)
That way we may also show what do we mean here and not only in the example.
what do you think 🤔?